data/json/monsters/insect_spider.json

```
@@ -1857,5 +1857,356 @@
+    "name": { "str": "giant aphid" },
+    "description": "A green aphid about the size of a rabbit.  It is constantly searching for plants to suck from.",
+    "default_faction": "ant",
+    "bodytype": "insect",
+    "species": [ "INSECT" ],
+    "volume": "1600 ml",
+    "weight": "2080 g",
+    "hp": 20,
+    "speed": 120,
+    "material": [ "iflesh" ],
+    "symbol": "a",
+    "color": "light_green",
+    "morale": 10,
+    "aggression": -90,
+    "melee_skill": 1,
+    "melee_dice": 1,
+    "melee_dice_sides": 2,
+    "melee_cut": 2,
+    "dodge": 3,
+    "armor_bash": 2,
+    "armor_cut": 5,
+    "armor_bullet": 3,
+    "vision_day": 10,
+    "vision_night": 5,
+    "harvest": "arachnid",
+    "regen_morale": true,
+    "fear_triggers": [ "FRIEND_ATTACKED", "FRIEND_DIED", "HURT", "FIRE", "PLAYER_CLOSE" ],
+    "death_function": [ "NORMAL" ],
+    "special_attacks": [ [ "EAT_FOOD", 60 ], [ "EAT_CROP", 80 ] ],
+    "reproduction": { "baby_monster": "mon_aphid", "baby_count": 1, "baby_timer": 20 },
+    "baby_flags": [ "SPRING", "SUMMER" ],
+    "biosignature": { "biosig_item": "honeydew", "biosig_timer": 600 },
+    "flags": [ "SEES", "HEARS", "SMELLS", "CLIMBS", "PATH_AVOID_FIRE", "PATH_AVOID_FALL" ]
+  },
+  {
+    "id": "mon_mantis_small",
+    "type": "MONSTER",
+    "name": { "str": "huge praying mantis" },
+    "description": "A green mutant insect with sharp grappling spikes on its front legs.  It's a little smaller than an adult person.",
+    "copy-from": "mon_mantis_giant",
+    "proportional": { "hp": 0.5, "speed": 1.1, "morale": 0.67, "melee_dice_sides": 0.67 },
+    "volume": "60 L",
+    "weight": "80 kg",
+    "melee_cut": 8,
+    "upgrades": { "half_life": 14, "into": "mon_mantis_giant" }
+  },
+  {
+    "id": "mon_mantis_giant",
+    "type": "MONSTER",
+    "name": { "str": "giant praying mantis" },
+    "description": "An enormous green creature grown about the size of a bear.  It dances weirdly and looks ready to chop your head off at any moment.",
+    "default_faction": "mantis",
+    "bodytype": "insect",
+    "species": [ "INSECT" ],
+    "volume": "120 L",
+    "weight": "160 kg",
+    "hp": 120,
+    "speed": 100,
+    "material": [ "iflesh" ],
+    "symbol": "M",
+    "color": "light_green",
+    "aggression": 50,
+    "morale": 80,
+    "melee_skill": 8,
+    "melee_dice": 1,
+    "melee_dice_sides": 6,
+    "melee_cut": 12,
+    "dodge": 3,
+    "armor_bash": 10,
+    "armor_cut": 16,
+    "armor_bullet": 13,
+    "vision_day": 15,
+    "vision_night": 5,
+    "harvest": "arachnid",
+    "anger_triggers": [ "PLAYER_WEAK", "PLAYER_CLOSE" ],
+    "fear_triggers": [ "FIRE" ],
+    "death_function": [ "NORMAL" ],
+    "special_attacks": [
+      [ "RANGED_PULL", 30 ],
+      [ "GRAB_DRAG", 10 ],
+      { "type": "leap", "cooldown": 8, "max_range": 5, "allow_no_target": false }
+    ],
+    "flags": [ "POISON", "SEES", "HEARS", "SMELLS", "CLIMBS", "PATH_AVOID_FIRE", "PATH_AVOID_FALL", "STUMBLES" ]
```

```suggestion
    "flags": [ "SEES", "HEARS", "CLIMBS", "PATH_AVOID_FIRE", "PATH_AVOID_FALL" ]
```
To make them a bit better as ambush predators; see below re POISON.